core/src/test/scala/org/apache/spark/MapOutputTrackerSuite.scala

```diff
@@ -855,7 +855,7 @@ class MapOutputTrackerSuite extends SparkFunSuite with LocalSparkContext {
     rpcEnv.shutdown()
   }
 
-  test("SPARK-37023: Avoid fetching merge status when shuffleMergeEnabled is false") {
+  test("SPARK-37023: Avoid fetching merge status when isShuffleMergeFinalizedMarked is false") {
```

Comment:
nit: isShuffleMergeFinalizedMarked -> useMergeResult ?
That is what we are testing here actually (this test went through some changes as code evolved, but looks like test name was missed).